ppb/systems.py

```diff
@@ -125,14 +125,17 @@ def activate(self, engine):
 
 class Renderer(System):
 
-    def __init__(self, resolution=default_resolution, **kwargs):
+    def __init__(self, resolution=default_resolution, window_title=None, **kwargs):
```

Comment:
Instead of `None`, let's default to `"PursuedPyBear"`. Bonus, we don't need the None check at L137.